playbooks/roles/edxapp/templates/private_requirements.txt.j2

```diff
@@ -0,0 +1,5 @@
+# This file was automatically generated by Open edX Configuration.
```

Comment:
@OmarIthawi @abeals @bryanlandia I figure you guys would have the strongest opinion here. Do you think this is ok or do you think we should change or add to the message?

Reply:
@johnbaldwin I think this is the standard way to generate that message: 

https://github.com/appsembler/configuration/blob/9e9a467f1cf0bfd8b06a926747d40ccd8f2a9d67/playbooks/roles/edx_service/templates/config.yml.j2#L2